lib/generators/components/templates/component_test.rb.erb

```diff
@@ -0,0 +1,11 @@
+require 'component_test_helper'
+
+class <%= @test_name %>Test < ComponentTestCase
```

Comment:
Fair point. So should I include them as templates and put them in place if they're not found? Or just look for them and output a warning? Seems like putting all this stuff in place could end up getting quite convoluted.